lib/controllers/dojo/save.js

```diff
@@ -59,13 +73,37 @@ module.exports = function (args, done) {
     seneca.act({role: 'cd-dojos', entity: 'dojo', cmd: 'save', dojo: dojo}, sCb);
   }
 
+  function notifyEvent (sCb) {
```

Comment:
Maybe `updateEventAddresses`? It took me a while to realise this isn't notifying users of an address change.

Reply:
fair, the idea was more about in the pub/sub logic, you "notify' the event service of a change

Reply:
If it was a proper pub/sub notify, it instead (I think) would just publish a `dojoAddressUpdated` event, and any service can do what they want with that. Here, we're specifically telling the events microservice to update its addresses for 'Dojo X'.

The former would be nice, but we don't really have the architecture in place to handle that. Kue would sort of work, but wouldn't allow for multiple consumers I believe? We don't strictly need multiple consumers (yet), but it'd be something I'd look for in this kind of set up personally.

Reply:
we'd need to replace http transport w/ amqp or eq.
It's planned, in my head, somewhere